rust-toolchain.toml

```diff
@@ -0,0 +1,6 @@
+[toolchain]
+# We should use stable here once nitro-testnode is updated and the contracts fit
```

Comment:
@qalisander should we add a GH issue for this?

Reply:
We already have [it](https://github.com/OpenZeppelin/rust-contracts-stylus/issues/129). Duplicated it in comment.